### ISSUE TITLE
Accents in files should be supported.

### ISSUE THREAD

**Issue by ericxuo** [eteran/nedit-ng#31]
Hi,

When I want to write a text file in french, accentuated letters are not supported (this was already the case with the native nedit).
Ex : I want to write 'é' or 'à'.
I get what is displayed in the attached file.
Is there any solution for this issue ?

Regards.

Xuo.

![accents](https://user-images.githubusercontent.com/9089499/36643839-536b448e-1a51-11e8-861e-df13586d68eb.png)

**Comment by eteran** [comment]
If I understand you correctly, you are saying that **both** nedit and nedit-ng do not support certain characters that are outside of the ASCII character range? I definitely plan to add things like UTF-8 support, but it will not be a small change, so it will take some time in both design and implementation.

**Comment by ericxuo** [comment]
Hi,
Yes, this is what I say. Both nedit and nedit-ng do not support (at least) accentuated characters, and probably others.
Xuo.

**Comment by eteran** [comment]
Right. At the moment, text in the editor view is rendered as if it were using Latin-1. Adding generalized support for Unicode will be very complicated.

For example, when the user simply clicks in the editor, a non-trivial amount of work needs to be done to figure out which character they clicked on since it isn't just a simply byte offset anymore. I plan to get there eventually, but it is going to be complicated :-).

**Comment by ericxuo** [comment]
Hi,

Here is a fork (??) of nedit which has implemented UTF-8 support (https://github.com/unixwork/xnedit). This may help you in case you still think about it :smile:

Regards.

Xuo.
